Rakefile

```diff
@@ -10,8 +11,7 @@ TEST_WITH_OLD_AND_NEW_API = %w[
 ].freeze
 
 def dry_v_test_files
-  dry_v_version = ENV.fetch("DRY_VALIDATION", "~> 0.13.0")
-  api = dry_v_version.gsub("~>", "").to_f >= 1.0 ? "new" : "old"
+  api = Gem::Version.new(Dry::Types::VERSION).to_s.split('.').first.to_i >= 1 ? "new" : "old"
```

Comment:
Do we need to check `Dry::Types` or `Dry::Validation` version here ?

Reply:
It’s the same - this is an horrible hack that I have done to keep both old/new api test and compatibility which we will drop soon. No need to spend any more time on it - happy if @seuros wants to change it to dry_validation but yeah not a big deal 👍🏼